### ISSUE TITLE
yajiedesign/mxnet "This branch is 137 commits behind apache:master."

### ISSUE THREAD

**Issue by Pagey** [apache/incubator-mxnet#16070]
does anybody know why this branch hasn't been updated in a month?  @yajiedesign can it be fixed? new precompiled releases are produced every day but on outdated source

Thanks

**Comment by haojin2** [comment]
@Pagey Are you trying to use @yajiedesign's personal fork?

**Comment by Pagey** [comment]
Yep the windows precompiled builds- they are also referenced in MXNet windows docs- the files are compiled (automatically?) about once a day, but it like the fork hasn’t been updated from mxnet master since 1st August 2019

**Comment by leleamol** [comment]
@mxnet-label-bot add [Doc, Build, Windows]

**Comment by haojin2** [comment]
@Pagey Would you please kindly point me to the doc that you're using now? so that I could double-check and provide help to you.

**Comment by Pagey** [comment]
@haojin2 e.g. here: https://mxnet.incubator.apache.org/versions/master/install/windows_setup.html
as i said, the prebuilds are build every day or other day there, but for some reason the code hasn't been updated in a month- me and others use these prebuilds for windows installations- i could compile myself from source but this was better

**Comment by yajiedesign** [comment]
Some problems with "git pull" in building system.The problem has been fixed, but it will be suspended for a few days because of some other problems.
